tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -462,13 +481,18 @@ vi_mean = vec(mean(z; dims=2))[[
 ```
 
 ```julia
-mean(chain).nt.mean
+mcmc_mean = mean(chain).nt.mean
```

Comment:
This seems a bit strange, I don't think this is how users are supposed to compute the mean.

Reply:
yes agree, we can use `chain[:intercept]` etc to extract the sample trajectories, however, we have 13 variables and it would be less elegant to do so. Any quick way maybe?

Reply:
Maybe something like
```suggestion
mcmc_mean = mean(chain, names(chain, :parameters))
```
(untested)

Reply:
Yes, tested as follows:
```
mean(chain, names(chain, :parameters))[:, 2]
```